src/Crypto/KmsMaterialsProvider.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
```

Comment:
Very nit but passim: I believe SDK style requires no newline between the file opener and the namespace declaration.